parts/header.html

```diff
@@ -1,7 +1,7 @@
 <!-- wp:group {"layout":{"inherit":"true"}} -->
-<div class="wp-block-group">
-	<!-- wp:group {"align":"full","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"bottom":"var(--wp--preset--spacing--50)","top":"var(--wp--preset--spacing--50)"}}}} -->
-	<div class="wp-block-group alignfull" style="padding-bottom:var(--wp--preset--spacing--50);padding-top:var(--wp--preset--spacing--50)">
+<div class="wp-block-group" style="display: flex;flex-direction: row;justify-content: center;">
```

Comment:
We should consider using inline styles as a bad practice and stay away from them. We should try to control as many styling things in `theme.json` as possible. Again though, I would focus on just the padding in the header (or maybe not even that) and not worry about addressing anything else as there will be a separate header card.

Reply:
Removed header changes, if it's gonna be totally changed anyway, no need for more conflicts
d0002bf01ba5c02f29e3026e329668bcb4fc47c0